### ISSUE TITLE
Question about the MSA datasets

### ISSUE THREAD

**Issue by dohyeonscottkim** [bytedance/Protenix#30]
Hello, thank you for such great work!

I’m interested in testing your model with additional MSA results.

Could you explain why the MSA pairing dataset was changed from `uniprot` to `uniref100`?

Also, what dataset did you use for the non-pairing dataset? Is it the same as in AlphaFold 3, such as `uniref90`, `small_bfd`, or `mgnify`?

**Comment by Kqiii** [comment]
Hi @dohyeonscottkim, we tested the MSA data pipeline described in the AF3 supplementary information as part of our pilot experiments, using the databases and search tools outlined. However, we found that scaling this pipeline for large-scale MSA searches was too time-consuming given our available computational resources. As a result, we decided to adopt the MSA pipeline used in [ColabFold](https://github.com/sokrypton/ColabFold/tree/main), which is significantly more efficient.
